### DISCUSSION TITLE
Chroma Key?

### DISCUSSION THREAD

**Discussion by invishane** [keiwando/toonsquid#503] [Ideas]
I’m finding that I can almost edit an entire video within TS which is awesome but sometimes I would love the ability to bring in a green screen clip - or if I export an animation with green screen be able to key it so I can re-use - I’m new to toonsquid so go easy on me! LOL

**Comment by kavlor** [comment]
There are already apps that you can do this with like Lumafusion and just export as Mov with transparent background which TS supports. Removing green from video is complex where you get different shades and hues of green which Lumafusion is specialised for and has different dials to tweak it plus there is the issue of doing it in realtime which Lumafusion is also specialised for. This is more of video processing feature rather than an animation feature so it is best done first else it is just slowing things down and hogging memory.

**Reply by invishane** [reply]
i get it lol. I use adobe premeire pro for things like that super easy on the pc. it's just trying to avoid extra steps it would be stress saving :)
but thank you for your advice i will look up and check out Lumafusion, i had not heard of that software appreciate it!
